service/ziti-tunnel/cli/service.go

```diff
@@ -53,3 +70,47 @@ func SetLogLevel(args []string, flags map[string]bool) {
 func GetFeedback(args []string, flags map[string]bool) {
 	GetDataFromMonitorIpcPipe(&dto.FEEDBACK_REQUEST, args, flags)
 }
+
+func UpdateConfigIPSubnet(args []string, flags map[string]interface{}) {
+	CIDRstring := flags["CIDR"]
+	AddDns := flags["AddDns"]
+	log.Info("Updating the config file")
+	var cidr []string
+	var ipMask int
+	var err error
+
+	updateTunIpv4Payload := make(map[string]interface{})
+	if CIDRstring != "" {
+		cidr = strings.Split(CIDRstring.(string), "/")
```

Comment:
just use `net.ParseCIDR` and `net.ParseIP` sdk.go and you don't need this stuff and it's more reliable

```

	ip, ipnet, err := net.ParseCIDR(hostOrIpOrCidr)
	if err != nil {
		//must not be an CIDR... try IP parse
		ip = net.ParseIP(hostOrIpOrCidr)
```